### PR TITLE
Reorganized project for packaging and distribution over pip

### PR DESCRIPTION
**Pull request by agraubert**
Changed directory structure and package names to better organize for pip packaging.

pVAC-Seq is currently packaged and uploaded to the pypi test site
you can install the package with
`pip install -i https://testpypi.python.org/simple --no-cache-dir pvacseq`
and then run `pvacseq` from the command line

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    classifiers=[
+        'Development Status :: 4 - Beta',
+
+        'Intended Audience :: Science/Research',
+        'Topic :: Scientific/Engineering :: Bio-Informatics',
+
+        "Programming Language :: Python :: 3.5"
+    ],
+
+    author = "Griffith Lab",
```
Comment (susannasiebert) [inline setup.py]: This should be updated to "Jasreet Hundal, Susanna Kiwala, Aaron Graubert, Jason Walker, Chris Miller, Malachi Griffith and Elaine Mardis"

**Comment by susannasiebert** [comment]
+1 with updated author list. @chrisamiller @jhundal could you please review and +1 this PR?

**Comment by agraubert** [comment]
Apparently none of the packaging libraries for python support specifying a python version, so I've added a workaround.  When installing, the package should print an error and halt if the version is lower than 3.5
There's currently a PEP open for extending the functionality of the packaging libraries (including adding python version requirements), but it hasn't been implemented yet

**Comment by acoffman** [comment]
+1

**Pull request merged**